<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RefDialog</class>
 <widget class="QDialog" name="RefDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Reference</string>
  </property>
  <widget class="QPushButton" name="pushBtn_ok">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtn_cancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Display:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>35</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Link:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_display">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_link">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
